src/blockchain/wallet.ts

```diff
@@ -23,43 +23,50 @@ export const accepted$ = interval(500).pipe(
   distinctUntilChanged(isEqual)
 );
 
-export const connectToWallet$: Subject<number> = new Subject();
-export const walletStatus$: Observable<WalletStatus> = merge(
-  combineLatest(
-    account$,
-    accepted$
-  ).pipe(
-    map(([account, hasAcceptedToS]) =>
+const connectToWallet$: Subject<number> = new Subject();
+
+export function connectToWallet() {
+  connectToWallet$.next(1);
```

Comment:
Why do you provide 1? It really doesn't matter as far as I know.

Reply:
Also why do we need to provide type to the Subject?

Reply:
No need to expose subjects all over the place.